sdcm/nemesis.py

```diff
@@ -1813,6 +1833,14 @@ def disrupt(self):
         time.sleep(300)
 
 
+class GrowShrinkClusterNemesis(Nemesis):
+    disruptive = True
```

Comment:
This nemesis isn't disruptive by itself.
However, if a disruptive nemesis will run parallel to this, it will fail this nemesis and may leave the cluster in  unwanted state. (too many nodes for example).
Hence it's good to mark it as disruptive. (maybe we should note it somewhere)

Reply:
As we discussed with Bentsi, this allow to run in Parallelnemesis tests

Reply:
@roydahan how can it be disruptive when running in parallel?
it is first adding an additional node  and then deleting it

Reply:
actually thinking of this again, I think Syclla doesn't support adding two nodes in parallel